src/gui/uithememanager.cpp

```diff
@@ -56,9 +59,15 @@ void UIThemeManager::initInstance()
 UIThemeManager::UIThemeManager()
 {
     const Preferences *const pref = Preferences::instance();
-    if (pref->useCustomUITheme()
-        && !QResource::registerResource(pref->customUIThemePath(), "/uitheme"))
-        LogMsg(tr("Failed to load UI theme from file: \"%1\"").arg(pref->customUIThemePath()), Log::WARNING);
+    if (pref->useCustomUITheme()) {
+        if (!QResource::registerResource(pref->customUIThemePath(), "/uitheme")) {
+            LogMsg(tr("Failed to load UI theme from file: \"%1\"").arg(pref->customUIThemePath()), Log::WARNING);
+        }
+        else {
+            loadColorsFromJSONConfig();
+            loadCustomPalette();
```

Comment:
I would still rename it to `applyPalette` to differentiate the functions that "loads something from config files" from the functions that "apply some settings to application".